molfeat/trans/pretrained/hf_transformers.py

```diff
@@ -424,7 +430,7 @@ def _embed(self, inputs, **kwargs):
             hidden_state = out_dict["hidden_states"]
             emb_layers = []
             for layer in self.concat_layers:
-                emb = hidden_state[layer].detach().cpu()  # B, S, D
+                emb = hidden_state[layer].detach().to(self.device)  # B, S, D
```

Comment:
I think this should already be on the device.

Reply:
Please confirm, then I would merge.

Reply:
I'll have some time to confirm in the next hour. thanks!

Reply:
you're correct, pushed the change